Troubleshooting: A drop-down editor does not show data (#4481)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/15 DataField and ValueExpr Should Be the Same Type.md

> If you use a lookup column editor in the DataGrid or TreeList, you need to specify the [columns](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/).[dataField](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#dataField) property. The [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/).[key](/Documentation/ApiReference/Data_Layer/ArrayStore/Configuration/#key), the [lookup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/).[valueExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/#valueExpr) and the [columns](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/).[dataField](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#dataField) values should be the same data type. In the code below, the `StateID` and `ID` fields should have the same type (for example, `String`).

Reviewer: We already stated above that `store.key` should match `lookup.valueExpr`. So we can remove `store.key` from the list here and simplify the description?

```suggestion
If you use a lookup column editor in the DataGrid or TreeList, you need to specify the [columns](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/).[dataField](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#dataField) property. The data type in that field must match the type of values in [lookup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/).[valueExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/#valueExpr). In the code below, the `StateID` and `ID` fields should have the same type (for example, `String`).
```
